docs/rules/semi.md

```diff
@@ -66,10 +66,15 @@ String option:
 * `"always"` (default) requires semicolons at the end of statements
 * `"never"` disallows semicolons as the end of statements (except to disambiguate statements beginning with `[`, `(`, `/`, `+`, or `-`)
 
-Object option:
+Object option (when `"always"`):
 
 * `"omitLastInOneLineBlock": true` ignores the last semicolon in a block in which its braces (and therefore the content of the block) are in the same line
 
+Object option (when `"never"`):
+
+* `"beforeUnreliableLineButSafe": "always"` requires semicolons at the end of statements if the next line starts with `[`, `(`, `/`, `+`, or `-`.
+* `"beforeUnreliableLineButSafe": "never"` disallows semicolons as the end of statements if it doesn't make ASI hazard even if the next line starts with `[`, `(`, `/`, `+`, or `-`.
```

Comment:
Can you clarify the default value of `beforeUnreliableLineButSafe` when `never` is used? (Is it `"ignore"`? If so, should we add an explicit `"ignore"` option?)

Reply:
OK.

I was hovering between adding `"any"` or not because similar `omitLastInOneLineBlock` option does not have the description about its default value.